barretenberg/cpp/src/barretenberg/stdlib/test_utils/tamper_proof.hpp

```diff
@@ -75,10 +79,21 @@ void tamper_with_proof(InnerProver& inner_prover, ProofType& inner_proof, Tamper
         break;
     }
     }
+
+    // Serialize transcript
+    // As inner_proof is extracted with export_proof, the internal values of inner_prover.transcript are reset
+    // Therefore, if we were to call export_proof without overriding num_frs_written and proof_start, the proof would
+    // be empty. This is a hack, we should probably have a better way of tampering with proofs.
```

Comment:
You could link this issue here: https://github.com/AztecProtocol/barretenberg/issues/1411. (Typical format is` // TODO(https://github.com/AztecProtocol/barretenberg/issues/1411): <brief description>`)